php/php.editor/src/org/netbeans/modules/php/editor/verification/PHP54UnhandledError.java

```diff
@@ -156,7 +156,7 @@ public void visit(Scalar node) {
             if (CancelSupport.getDefault().isCancelled()) {
                 return;
             }
-            if (node.getScalarType().equals(Scalar.Type.REAL) && node.getStringValue().startsWith(BINARY_PREFIX)) {
+            if (node.getScalarType().equals(Scalar.Type.FLOAT) && node.getStringValue().startsWith(BINARY_PREFIX)) {
```

Comment:
@junichi11 Again, please, verify that we can simply replace `REAL` -> `FLOAT` and not keep the both types. Thank you.

Reply:
It should be OK (I assume all `REAL` types were replaced with `FLOAT` in the parser).
But I would keep both types, just in case.

Reply:
The only use case would be if we had it in the index, right? Otherwise, we should be safe.

But as we agreed, I believe that keeping both types should be super-safe. Once we remove the `REAL` type, it will be very easy to delete all its usages.

@troizet, could you, please, update this PR? Thanks for understanding, and sorry for any inconveniences, we are trying to be really safe here.